aspnetcore/fundamentals/w3c-logger/index.md

```diff
@@ -49,67 +49,10 @@ By default, W3CLogger logs common properties such as path, status-code, date, ti
-| `UriQuery` |  Request <xref:Microsoft.AspNetCore.Http.HttpRequest.QueryString>. | 0x200 |
-| `ProtocolStatus` |  Response <xref:Microsoft.AspNetCore.Http.HttpResponse.StatusCode>. | 0x400 |
-| `TimeTaken` |  The duration of time, in milliseconds, that the action consumed. | 0x800 |
-| `ProtocolVersion` |  Request <xref:Microsoft.AspNetCore.Http.HttpRequest.Protocol>. | 0x1000 |
-| `Host` |  Request `Microsoft.AspNetCore.Http.Headers.HeaderNames.Host`. | 0x2000 |
-| `UserAgent` |  Request `Microsoft.AspNetCore.Http.Headers.HeaderNames.UserAgent`. | 0x4000 |
-| `Cookie` |  Request `Microsoft.AspNetCore.Http.Headers.HeaderNames.Cookie`. | 0x8000 |
-| `Referer` |  Request `Microsoft.AspNetCore.Http.Headers.HeaderNames.Referer`. | 0x10000 |
-| `ConnectionInfoFields` | Flag for logging a collection of properties, about the HTTP Connection, including `ClientIpAddress`, `ServerIpAddress`, and `ServerPort`. | `ClientIpAddress | ServerIpAddress | ServerPort` |
-| `RequestHeaders` | Flag for logging a collection of request headers, including `Host`, `Referer`, and `UserAgent`. | `Host | Referer | UserAgent` |
-| `Request` | Flag for logging a collection of properties about the request, including `UriStem`, `UriQuery`, `ProtocolVersion`, `Method`, and `RequestHeaders`. | `UriStem | UriQuery | ProtocolVersion | Method | RequestHeaders` |
-| `All` | Flag for logging all possible fields. | `Date | Time | ServerName | Method | UriStem | UriQuery | ProtocolStatus | TimeTaken | ProtocolVersion | Host | UserAgent | Referer | ConnectionInfoFields | UserName | Cookie` |
-
-### `FileSizeLimit`
-
-Maximum log file size in bytes. Defaults to 10 MiB.
-
-[!code-csharp[](samples/6.x/Program.cs?name=configureservices&highlight=6)]
-
-### `RetainedFileCountLimit`
-
-Maximum number of files to keep on disk before rolling, per application. Defaults to 4, capped at 10,000.
-
-[!code-csharp[](samples/6.x/Program.cs?name=configureservices&highlight=7)]
-
-### `FileName`
-
-Prefix to be used for log file name. The current date plus a file number, in the format `{YYYYMMDD.X}`, will be appended.
-
-[!code-csharp[](samples/6.x/Program.cs?name=configureservices&highlight=8)]
-
-### `LogDirectory`
-
-Directory where the log file will be written to. Defaults to `./logs/`, relative to the app directory.
-
-[!code-csharp[](samples/6.x/Program.cs?name=configureservices&highlight=9)]
-
-### `FlushInterval`
-
-The period after which logs will be flushed to the log file. Defaults to 1 second.
-
-[!code-csharp[](samples/6.x/Program.cs?name=configureservices&highlight=10)]
+<xref:Microsoft.AspNetCore.HttpLogging.W3CLoggerOptions.LoggingFields%2A?displayProperty=nameWithType> is a bit flag enumeration that configures specific parts of the request and response to log, and other information about the connection. `LoggingFields` defaults to include all possible fields except `UserName` and `Cookie`. 
```

Comment:
Can you link to https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.httplogging.w3cloggingfields
so they get all the fields?

Reply:
Great suggestion - done. Feel free to merge if you're happy with the addition.